schema-next.yaml

```diff
@@ -12,6 +12,7 @@ versions:
         # https://github.com/open-telemetry/semantic-conventions/pull/1383
         - rename_attributes:
             attribute_map:
+              gen_ai.openai.request.seed: gen_ai.request.seed
```

Comment:
sorry, I mislead you with my prev comment, please add 

```yaml
    all:
      changes:
        - rename_attributes:
            attribute_map:
              # https://github.com/open-telemetry/semantic-conventions/pull/1683
              gen_ai.openai.request.seed: gen_ai.request.seed
```

under `next:` (the current version, 1.29.0 is already released)

Reply:
Done, I should always update next, thanks @lmolkova !